cmd/argo/commands/server.go

```diff
@@ -103,5 +105,6 @@ See %s`, help.ArgoSever),
 	command.Flags().StringVar(&logLevel, "loglevel", "info", "Set the logging level. One of: debug|info|warn|error")
 	command.Flags().BoolVar(&namespaced, "namespaced", false, "run as namespaced mode")
 	command.Flags().StringVar(&managedNamespace, "managed-namespace", "", "namespace that watches, default to the installation namespace")
+	command.Flags().BoolVar(&disableOpenBrowser, "disable-open-browser", false, "disable automatic launching of the browser [Hosted mode]")
```

Comment:
could we add a env var for this, e.g. `ARGO_OPEN_BROWSER` so people could set this in their bash_profile if they want - i.e. default it on for themselves